test/github-issues/9687/entity/Foo.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, Column, PrimaryGeneratedColumn } from "../../../../src"
+
+@Entity({ name: "foo", schema: "SYSTEM" })
+export class Foo {
+    @PrimaryGeneratedColumn()
+    id: number
+
+    @Column({ name: "string", default: "string" })
+    string: number
+
+    @Column({ name: "number", default: 1 })
+    number: number
+
+    @Column({ name: "boolean", default: true })
+    boolean: number
+
+    @Column({ name: "date", default: Date.now })
```

Comment:
column type should be `timestamp`
`Date.now` should be truncated to fit javascript `number` size -> `Math.round(Date.now() / 1000)`